src/Symfony/Bridge/PhpUnit/DeprecationErrorHandler.php

```diff
@@ -162,9 +184,18 @@ public static function register($mode = 0)
                     return $b['count'] - $a['count'];
                 };
 
-                foreach (array('unsilenced', 'remaining', 'legacy', 'other') as $group) {
+                $groups = array('unsilenced', 'remaining');
+                if (DeprecationErrorHandler::MODE_WEAK_VENDORS === $mode) {
+                    $groups[] = 'remaining vendor';
+                }
+                array_push($groups, 'legacy', 'other');
+
+                foreach ($groups as $group) {
                     if ($deprecations[$group.'Count']) {
-                        echo "\n", $colorize(sprintf('%s deprecation notices (%d)', ucfirst($group), $deprecations[$group.'Count']), 'legacy' !== $group), "\n";
+                        echo "\n", $colorize(
+                            sprintf('%s deprecation notices (%d)', ucfirst($group), $deprecations[$group.'Count']),
+                            'legacy' !== $group && 'remaining vendor' !== $group
+                        ), "\n";
```

Comment:
I hope you have nothing about the multiline `$colorize` call. IMO it's more readable in an objective way (you can see all the code w/o scrolling when reviewing, for instance).